src/main/java/org/fundacionjala/coding/dennis/Movie.java

```diff
@@ -0,0 +1,73 @@
+package org.fundacionjala.coding.dennis;
+
+/**
+ *
+ */
+public abstract class Movie {
+    private String title;
+    private PriceCode priceCode;
+
+    /**
+     *
+     * @param title of the movie.
+     */
+    public Movie(final String title) {
+        this.title = title;
+    }
+
+    /**
+     *
+     * @return price.
+     */
+    public PriceCode getPriceCode() {
+        return getpriceCode();
+    }
+
+    /**
+     *
+     * @param arg price.
+     */
+    public void setPriceCode(final PriceCode arg) {
+        setpriceCode(arg);
+    }
+
+    /**
+     *
+     * @return title.
+     */
+    public String getTitle() {
+        return title;
+    }
+
+    /**
+     *
+     * @return price.
+     */
+    public PriceCode getpriceCode() {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename method "getpriceCode" to prevent any misunderstanding/clash with method "getPriceCode" defined on line 22. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1845)